lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -538,6 +553,67 @@ setNonEnumerableReadOnly( REPL.prototype, '_prompt', function prompt() {
 	return inputPrompt( this._inputPrompt, this._count );
 });
 
+/**
+* Resolves any existing REPL configuration.
+*
+* @private
+* @name _resolveConfiguration
+* @memberof REPL.prototype
+* @type {Function}
+* @returns {Object} configuration object
+*/
+setNonEnumerableReadOnly( REPL.prototype, '_resolveConfiguration', function resolveConfiguration() {
+	/* eslint-disable stdlib/no-dynamic-require, no-unused-vars */
+	var config;
+	var path;
+
+	path = resolveParentPath( '.stdlib_repl.json', {
+		'dir': cwd()
+	});
```

Comment:
I am confused as to how do we resolve a `.json` or a `.js` file? Is there a way using regexp?

Reply:
Use `resolve-parent-path-by`. https://github.com/stdlib-js/stdlib/tree/1cb0243d79c25ae193c058bb15ceeebad6cf3cfd/lib/node_modules/%40stdlib/fs/resolve-parent-path-by

Reply:
Correct me if I'm wrong, but doesn't it still take just one param for the path and the predicate validates that?

Reply:
You're right. Sorry. There isn't a way to use a regexp, as there would be ambiguity as to what to return when there are multiple matches.

Seems like we could use another utility package. E.g., `@stdlib/fs/resolve-parent-paths`, which instead of a single path, accepts either an array of paths or a regular expression. It should still return the first set of matches. Feel free to open an RFC and implement.